pycqed/instrument_drivers/physical_instruments/NanotecSMI33.py

```diff
@@ -0,0 +1,768 @@
+                       'Min value: 1'
+                       'Max value: 65535'))
+
+        self.add_parameter(
+            'acceleration_mode',
+            label='Acceleration Mode',
+            unit='',
+            get_cmd=':ramp_mode',
+            set_cmd=':ramp_mode={}',
+            max_val_age=0,
+            val_mapping={'Trapezoidal': 0,
+                         'Sinusoidal': 1,
+                         'Jerk-free': 2},
+            docstring='Set the acceleration mode')
+
+        self.add_parameter(
+            'acceleration_jerk',
+            label='Acceleration Jerk',
+            unit='',
+            get_cmd=':b',
+            set_cmd=':b{}',
+            get_parser=int,
+            vals=Ints(min_value=1,
+                      max_value=100000000),
+            docstring=('Acceleration jerk'
+                       'Min value: 1'
+                       'Max value: 100000000'))
+
+        self.add_parameter(
+            'braking',
+            label='Breaking',
+            unit='',
+            get_cmd='B',
+            set_cmd='B{}',
+            get_parser=int,
+            max_val_age=0,
+            vals=Ints(min_value=0,
+                      max_value=65535),
+            docstring=('Braking rate'
+                       'Min value: 1'
+                       'Max value: 65535'))
+
+        self.add_parameter(
+            'braking_jerk',
+            label='Braking Jerk',
+            unit='',
+            get_cmd=':B',
+            set_cmd=':B{}',
+            get_parser=int,
+            vals=Ints(min_value=1,
+                      max_value=100000000),
+            docstring=('Breaking jerk'
+                       'Min value: 1'
+                       'Max value: 100000000'))
+
+        self.add_parameter(
+            'continuation_record',
+            label='Continuation Record',
+            unit='',
+            get_cmd='N',
+            set_cmd='N{}',
+            get_parser=int,
+            vals=Ints(min_value=0,
+                      max_value=32),
+            docstring=('Record in EEPROM to continue with after finishing'
+                       'the current record.'
+                       'Min value is 0'
+                       'Max value is 32'
+                       'If the value is set to 0, this setting has no'
+                       'effect.'))
+
+        self.add_parameter(
+            'command_response',
+            label='Command Response',
+            unit='',
+            get_cmd=False,
+            set_cmd='|{}',
+            val_mapping={'Disabled': 0,
+                         'Enabled': 1},
+            docstring=('Enable or disable command response'
+                       'If disabled, the controller will obey all commands'
+                       'that are sent without responding.'
+                       'Note that this disables response to any commands'
+                       'including those reading out other settings'))
+
+        self.add_parameter(
```

Comment:
creating the the following parameters in a for-loop would make sense

Reply:
Done in the latest commit.